app/assets/scripts/map.js

```diff
@@ -369,11 +410,138 @@ function populateListDiv() {
+ *   marker
+ */
+function makeResponsive() {
+  // Change to a single column view
+  if ($(window).width() <= singleColNoSpaceBreakpoint) {
+    singleColumnResets();
+    $('#map-list-grid').removeClass('grid-space-large');
+    $('#map-list-grid').addClass('grid-space-small');
+  } else if (
+    $(window).width() > singleColNoSpaceBreakpoint
+    && $(window).width() <= singleColBreakpoint
+  ) {
+    singleColumnResets();
+    $('#map-list-grid').removeClass('grid-space-small');
+    $('#map-list-grid').addClass('grid-space-large');
+  } else if ( // Change to double column views
+    $(window).width() > singleColBreakpoint
+    && $(window).width() <= twoColResizeBreakpoint
+  ) {
+    $('#left-column').removeClass().addClass('five wide column');
+    $('#right-column').removeClass().addClass('eleven wide column');
+    doubleColumnResets();
+  } else if ($(window).width() > twoColResizeBreakpoint) {
+    $('#left-column').removeClass().addClass('four wide column');
+    $('#right-column').removeClass().addClass('twelve wide column');
+    doubleColumnResets();
+  }
+}
+
+// Switching from double -> single column
+// Default show list view
+function singleColumnResets() {
+  $('#left-column').removeClass().addClass('sixteen wide column');
+  $('#right-column').removeClass().addClass('sixteen wide column');
+  $('#right-column').hide();
+  setNavSwitching();
+}
+
+// Set a nav element that allows toggling between list and map view
+// when in single column mode
+function setNavSwitching() {
+  // initialize to 'Show Map'
+  $('#nav-to-list').hide();
+  $('#nav-to-map').show();
+
+  // Switch from list to map
+  $('#nav-to-map').click(function() {
+    listToMapSingleColumn();
+  });
+
+  // Switch from map to list
+  $('#nav-to-list').click(function() {
+    mapToListSingleColumn();
+  });
+}
+
+// Changing from list view to map view in single column mode
+function listToMapSingleColumn() {
+  $('#left-column').hide();
+  $('#right-column').show();
+  $('#map').show();
+  var center = map.getCenter();
+  // Need to call resize event on map or creates dead grey area on map
+  google.maps.event.trigger(map, "resize");
+  map.setCenter(center);
+  map.setZoom(14);
+
+  $('#nav-to-list').show();
+  $('#nav-to-map').hide();
+}
+
+// Changing from map view to list view in single column mode
+function mapToListSingleColumn() {
+  $('#left-column').show();
+  $('#right-column').hide();
+
+  $('#nav-to-list').hide();
+  $('#nav-to-map').show();
+}
+
+// Switching from single -> double column
+function doubleColumnResets() {
+  $('#map-list-grid').removeClass('grid-space-small');
+  $('#map-list-grid').removeClass('grid-space-large');
+
+  $('#map-footer').hide();
+  $('#left-column').show();
+  $('#right-column').show();
+  $('.nav-mobile-switch').hide();
 
   var center = map.getCenter();
   // Need to call resize event on map or creates dead grey area on map
   google.maps.event.trigger(map, "resize");
   map.setCenter(center);
+  map.setZoom(14);
```

Comment:
when a page is resized within the bounds of the desktop view, it zooms in too much & the center is determined in a different way from the default. try testing with resources far away from each other